plans/tier1.fmf

```diff
@@ -0,0 +1,42 @@
+discover+:
+    tier: 1
+prepare+:
+  - name: reboot machine
+    how: ansible
+    playbook: tests/ansible_collections/reboot.yml
+
```

Comment:
Bear in mind that these tests:

- "resolve-dependency" from "rhsm" conversion folder
- "one-kernel-scenarion" from "custom-repos" conversion folder

They need some extra preparation steps. Maybe we could move them to the tier0?